docs/gdr-s3-data-sharing.md

```diff
@@ -103,6 +103,7 @@ We need to specify which actions are allowed under the bucket permissions.
               "s3:PutObject",
               "s3:DeleteObject",
               "s3:PutObjectAcl",
+              "s3:PutObjectTagging",
```

Comment:
Perhaps allow all object tagging operations? Otherwise we may run into issues if we need to delete files?
`"s3:*Object*Tagging"`